controller/nodegroup.go

```diff
@@ -219,7 +219,10 @@ func createNodeGroup(config *eksv1.EKSClusterConfig, group eksv1.NodeGroup, eksS
 		}
 	}
 
-	launchTemplateVersion := aws.String(strconv.FormatInt(*lt.Version, 10))
+	var launchTemplateVersion *string
+	if lt.Version != nil {
```

Comment:
When would this be nil?

Reply:
It would only be `nil` if it was passed that way, but a better check has been added.